### PR TITLE
Statin cqm438

### PR DESCRIPTION
**Pull request by utsavpaudel**
## Describe your changes
Please include a summary of any changes.

## How has this been tested?
Please describe the tests you ran to verify your changes.  Provide instructions or code to reproduce output.

## Reviewer focus
Please summarize the specific items you’d like the reviewer(s) to look into.

## Checklist before requesting a review
- [ ] I have added at least one Github label to this PR (bug, enhancement, breaking change,...)
- [ ] My code follows [style guidelines](https://thetuvaproject.com/contribution-guides/development-style-guide)
- [ ] (New models) [YAML files](https://github.com/tuva-health/tuva/blob/main/models/hcc_suspecting/hcc_suspecting_models.yml) are categorized by sub folder and models listed in alphabetical order
- [ ] (New models) I have added a [config](https://github.com/tuva-health/tuva/blob/main/models/hcc_suspecting/final/hcc_suspecting__list.sql) to each new model to enable it for claims and/or clinical data
- [ ] (New models) I have added the variable `tuva_last_run` to the final output
- [ ] (Optional) I have recorded a Loom to explain this PR

### Package release checklist
- [ ] I have updated [dbt docs](https://www.notion.so/tuvahealth/Building-dbt-Docs-16df2f00df244f29b9d6756d8adbc2d9)
- [ ] I have updated the version number in the `dbt_project.yml`

### CODE REVIEW

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    cross join {{ref('quality_measures__int_cqm438__performance_period')}}
+
+)
+
+, ascvd_codes as (
+
+    select
+          code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql:117]: reference from earlier CTE

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+                when lower(source_code_type) = 'cpt' then 'hcpcs'
+                when lower(source_code_type) = 'snomed' then 'snomed-ct'
+                else lower(source_code_type)
+              end
+          ) as code_type
+        , coalesce (
+              normalized_code
+            , source_code
+          ) as code
+    from {{ ref('quality_measures__stg_core__procedure') }}
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql:176]: perform such transformations in earlier `procedure_encounters` cte and reference here

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    on evidence_date <= pp.performance_period_end
+
+)
+
+, qualifying_patients as (
+
+    select
+        distinct
+          patients_with_ascvd.patient_id
+        , patients_with_age.age as age
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_denominator.sql:229]: no need to alias here

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_exclusions.sql [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+
+)
+
+select
+      patient_id
+    , exclusion_date
+    , exclusion_reason
+    , exclusion_flag
+    , '{{ var('tuva_last_run')}}' as tuva_last_run
+from add_data_types
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_exclusions.sql:334]: add EOF

File: models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_long.sql [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+)
+
+, numerator as (
+
+    select
+          patient_id
+        , evidence_date
+        , evidence_value
+    from {{ ref('quality_measures__int_cqm438_numerator') }}
+    where numerator_flag = 1
```
Comment (deepsonshrestha) [inline models/quality_measures/intermediate/cqm438_statin_therapy_for_patients_with_cardiovascular_disease/quality_measures__int_cqm438_long.sql:29]: this redundant logic can be removed

**Review by deepsonshrestha — COMMENTED** [review]
Looks good 🚀

**Comment by utsavpaudel** [comment]
More submission criteria have been added to `statin123cr` branch, from which PR will be opened.

**Pull request closed without merging**